src/StatusQ/Components/StatusChatListAndCategories.qml

```diff
@@ -142,7 +142,14 @@ Item {
                         chatList.selectedChatId: statusChatListAndCategories.selectedChatId
                         chatList.onChatItemSelected: statusChatListAndCategories.chatItemSelected(id)
                         chatList.onChatItemUnmuted: statusChatListAndCategories.chatItemUnmuted(id)
-                        chatList.onChatItemReordered: statusChatListAndCategories.chatItemReordered(model.categoryId, id, from, to)
+                        chatList.onChatItemReordered: {
+                            console.log("REORDERING CHAT ITEM:")
+                            console.log("FROM CATEGORY: ", model.categoryId)
+                            console.log("TO: ", categoryId)
+                            console.log("FROM POSITION: ", from)
+                            console.log("TO POSITION: ", to)
```

Comment:
Obviously these need to be removed. Leaving them for now for debugging purpose. Will go away before merge